packages/react/src/components/Cta/Cta.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ArrowDown20, ArrowRight20, Launch20 } from '@carbon/icons-react';
+import { ButtonGroup } from '../../patterns/sub-patterns/ButtonGroup';
+import { CardLink } from '../../patterns/sub-patterns/CardLink';
+import { FeaturedLink } from '../../patterns/blocks/FeaturedLink';
+import { LinkWithIcon } from '../LinkWithIcon';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * CTA component
+ *
+ * @param {object} props props object
+ * @param {string} props.style style ( text | card | button | feature ).
+ * @param {object} props.type tyle ( jump | local | external ).
+ * @param {object} props.cta cta object which includes href,text, copy, card and image, button  properties.
+ * @returns {*} CTA Component
+ */
+const CTA = ({ style, type, ...cta }) =>
+  style === 'card' ? (
+    <CardLink {...cta} icon={iconSelector(type)} />
+  ) : style === 'button' ? (
+    <ButtonGroup buttons={renderButtons(cta)} />
+  ) : style === 'feature' ? (
+    <FeaturedLink
+      {...cta}
+      //icon={iconSelector(type)}
+    />
+  ) : (
+    <LinkWithIcon href={cta.href} target={cta.target}>
```

Comment:
This may be like this because you're still working on it, but I just want to reiterate in case there is a misunderstanding that the `target` here shouldn't always be here, we only need to define if the `type==="jump"`, and we need to apply it to all of the other ctas as well.

Reply:
Yeah @photodow still working on it.